web/react/components/command_list.jsx

```diff
@@ -44,6 +46,30 @@ export default class CommandList extends React.Component {
             this.props.channelId,
             cmd,
             true,
+            false,
+            function success(data) {
```

Comment:
You could use es6 fat arrow function `() => {}` this way you don't have to manually bind it (should probably be changed in `getSuggestedCommands` too)